assets/cart.js

```diff
@@ -53,6 +53,7 @@ class CartItems extends HTMLElement {
         const html = new DOMParser().parseFromString(responseText, 'text/html');
         const sourceQty = html.querySelector('cart-items');
         this.innerHTML = sourceQty.innerHTML;
+        document.querySelector('cart-items').classList.remove('is-empty');
```

Comment:
Since this is all within the declaration fo the custom element `cart-iems/CartItems`, you can use `this` instead:
```suggestion
        this.classList.remove('is-empty');
```

Reply:
Thanks for the review @ludoboludo 

I cleaned up the JS syntax in another commit per your suggestion. Regarding the drawer failing to update quantity, it's currently the same behaviour without the changes. I did notice that in my testing as well but figured it might be better suited as another issue, if it's needed at all. The drawer doesn't have an option to proceed to the cart page, so I think it's unlikely this use case will ever come up if the cart drawer is enabled.